seatunnel-common/src/main/java/org/apache/seatunnel/common/config/Common.java

```diff
@@ -47,6 +52,21 @@ private Common() {
 
     private static boolean STARTER = false;
 
+    /**
```

Comment:
Please don't add it in here, this `ADD_URL_TO_CLASSLOADER` only work for flink, spark use default `ADD_URL_TO_CLASSLOADER` in `AbstractPluginDiscovery`. So this method not 'common'